route/parse_new.go

```diff
@@ -87,6 +90,42 @@ func Parse(in string) (defs []*RouteDef, err error) {
 	return defs, nil
 }
 
+func ParseAliases(in string, cfg *config.Config) (names []string, err error) {
+	var defs []*RouteDef
+	var def *RouteDef
+	for i, s := range strings.Split(in, "\n") {
+		def, err = nil, nil
+		s = strings.TrimSpace(s)
+		switch {
+		case reComment.MatchString(s) || reBlankLine.MatchString(s):
+			continue
+		case reRouteAdd.MatchString(s):
+			def, err = parseRouteAdd(s)
+		default:
+			err = errors.New("syntax error: 'route' expected")
+		}
+		if err != nil {
+			return nil, fmt.Errorf("line %d: %s", i+1, err)
+		}
+		defs = append(defs, def)
+	}
+
+	var aliases []string
+
+	if cfg.Registry.Backend == "consul" && cfg.Registry.Consul.Register {
```

Comment:
Why is that important here? I'd just parse the aliases and let the backend decide what to do with them.